mirgecom/euler.py

```diff
@@ -352,6 +352,32 @@ def get_inviscid_cfl(discr, eos, dt, q):
     return dt / wanted_dt
 
 
+# By default, run unitless
+NAME_TO_UNITS = {
+    "mass": "",
+    "energy": "",
+    "momentum": "",
+    "temperature": "",
+    "pressure": ""
+}
+
+
+def units_for_logging(quantity: str) -> str:
+    """Return unit for quantity."""
+    return NAME_TO_UNITS[quantity]
```

Comment:
I see why we're doing this for our project.  I question whether the logger utility really needs this interface, however.  Can't the user define the units when they identify watched quantities?  

This could be misguided... but IMO, it is outside the scope of the logger to have knowledge of the physical content of the quantities it is watching.

Reply:
Sure, a user would just need to define their own `units_for_logging` function and pass that to the initialization. See e.g. https://github.com/illinois-ceesd/mirgecom/blob/ff9e2ea031794940fbf1352273489eebadc39ca0/examples/vortex-mpi.py#L72-L74